package.json

```diff
@@ -69,7 +69,7 @@
 		"rollup-plugin-version-injector": "^1.3.3",
 		"ts-jest": "^28.0.5",
 		"typedoc": "^0.22.18",
-		"typedoc-plugin-djs-links": "^1.1.0",
+		"typedoc-plugin-djs-links": "^1.0",
```

Comment:
You can specify 1.0.4 as a more exact version

Reply:
i thought i kept 1.0.4 oh wait